project-koku/koku
# [COST-1030] Fix data deletion for data in S3

## Summary
This fixes a bug where source deletion was not deleting S3 data. 

## Testing
1. Set envs

```
ENABLE_PARQUET_PROCESSING=True
S3_BUCKET_NAME=koku-bucket
S3_ENDPOINT=http://kokuminio:9000
S3_ACCESS_KEY=kokuminioaccess
S3_SECRET=kokuminiosecret
```

2.  make docker-up-min-presto
3. make create-test-customer
4. make load-test-customer-data start=2021-02-01 end=2021-02-15
5. Remove source
6. Check that source is no longer in minio
